### DISCUSSION TITLE
Approved for GitHub Student Developer Pack but Benefits Not Accessible After 3 Days

### DISCUSSION THREAD

**Discussion by stekatag** [community/community#140098] [GitHub Education] [answered]
### Hi everyone,

I was approved for the GitHub Student Developer Pack three days ago, but I still don't have access to the benefits that come with it. I understand that this process can take up to five days or even more, but I wanted to raise this discussion in hopes that someone might be able to help speed up the process.

I've checked my account, but I can't seem to use any of the offers included in the pack. I'm eager to start using the benefits for my projects and studies.

Is there anything else I should do, or is this delay normal? Any assistance or advice would be greatly appreciated.

Thank you!

![msedge_QJaWph71h4](https://github.com/user-attachments/assets/a0ce2f50-5e04-47e7-9ed0-47618c7e4317)

**Comment by davevad93** [comment, marked as answer]
Hi @stekatag ,

Don't worry it's normal, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @stekatag was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
